packages/Paragraph/package.json

```diff
@@ -21,14 +21,14 @@
   },
   "homepage": "http://tds.telus.com",
   "peerDependencies": {
-    "react": ">=15",
-    "react-dom": ">=15"
+    "react": "^16.8.2",
+    "react-dom": "^16.8.2",
+    "styled-components": "^4.1.3"
   },
   "dependencies": {
+    "@tds/core-colours": "^1.1.2",
```

Comment:
`core-colours` don't seem to be used anywhere in Paragraph. Should they be removed from `dependencies`?